src/utils/path.ts

```diff
@@ -218,3 +218,31 @@ export function relative(from: string, to: string): string {
 export function join(...paths: string[]): string {
   return normalizePath(path.join(...paths), false);
 }
+
+/**
+ * A wrapped version of node.js' {@link path.resolve} which adds our custom
+ * normalization logic. This resolves a path to a given (relative or absolute)
+ * path.
+ *
+ * @throws the underlying node function will throw if any argument is not a
+ * string
+ * @param paths a path or path fragments to resolve
+ * @returns a resolved path!
+ */
+export function resolve(...paths: string[]): string {
+  return normalizePath(path.resolve(...paths));
```

Comment:
Do these calls need the `false` argument that the others (`relative`, `join`) have?

Reply:
haha p sure I was adding that as you wrote this comment - they do, I just added it